app/lib/services.rb

```diff
@@ -30,4 +30,10 @@ def self.worldwide_api
   def self.registries
     Registries::BaseRegistries.new
   end
+
+  if Rails.configuration.feature_flag_govuk_accounts
+    def self.accounts_api
+      ENV.fetch("PLEK_SERVICE_ACCOUNT_MANAGER_URI")
```

Comment:
Did we add account-manager to PLEK already then?

Reply:
I don't think it needs adding to Plek, since Plek will guess at the URL (basically `app-name.env.publishing.service.gov.uk` or `app-name.govuk-internal.digital`) unless there are `PLEK_` environment variables set for the app, in which case it'll return the value for the environment variable.  You should be able to test this locally by opening a Rails console for `finder-frontend` and see it return the value you've set in the environment variable when executing `Plek.find("account-manager")`.

Reply:
ah ha, interesting, will test that now!

Reply:
It works! Cheers. Do you think i should remove the if statement for the feature flag too then?
I only had that there becuase I wanted to force devs to notice if they forget to set `PLEK_SERVICE_ACCOUNT_MANAGER_URI`. This way, it's impossible for them not to!

Reply:
I'd leave the if statement in place, then this code will only get executed if the feature is switched on (so there won't be an `accounts_api` service if accounts aren't switched on).

Reply:
Amended!